static/app/components/replays/table/replayTableColumns.tsx

```diff
@@ -494,56 +550,44 @@ export const ReplaySessionColumn: ReplayTableColumn = {
       });
 
     return (
-      <Flex key="session" align="center" gap={space(1)}>
-        <UserAvatar
-          user={{
-            username: replay.user?.display_name || '',
-            email: replay.user?.email || '',
-            id: replay.user?.id || '',
-            ip_address: replay.user?.ip || '',
-            name: replay.user?.username || '',
-          }}
-          size={24}
-        />
-        <SubText>
-          <Flex gap={space(0.5)} align="flex-start">
-            <DisplayNameLink
-              data-underline-on-hover
-              to={
-                isIssuesReplayList
-                  ? // if on the issues replay list, don't redirect to the details tab. this causes URL flickering
-                    {
-                      pathname: location.pathname,
-                      query: location.query,
-                    }
-                  : detailsTab()
-              }
-              onClick={trackNavigationEvent}
```

Comment:
The link stuff is simplified here. 

Now, because of the z-index layering, this column doesn't need to know about the IssuesReplayList. When we're on the IssueReplayList the PlayPause column will capture all the clicks and nothing will come into here.